nixos/modules/services/misc/etebase-server.nix

```diff
@@ -105,30 +114,27 @@ in
         };
       };
 
-      customIni = mkOption {
-        type = with types; nullOr str;
-        default = null;
+      extraConfig = mkOption {
```

Comment:
It's better to call this `settings` over `extraConfig` in line with <https://github.com/NixOS/rfcs/pull/42>. We are slowly migrating towards that style of module if possible.